src/client/components/simple-table/simple-table.tsx

```diff
@@ -72,6 +73,13 @@ export class SimpleTable extends React.Component<SimpleTableProps, SimpleTableSt
     return horizontalScrollShadowStyle;
   }
 
+  componentDidUpdate() {
+    if (!!this.props.onSizeChange) {
+      let bodyWidth = (this.refs['body-cont'] as any).getBoundingClientRect().width;
+      this.props.onSizeChange({bodyWidth});
```

Comment:
this backwards information flow really bothers me. Good thing it is not needed (see below)